StreamNode/Services/Settings/SettingsService.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Config.Net;
+using System.IO;
+using Newtonsoft.Json;
+
+namespace StreamNode.Services.Settings
+{
+    public class SettingsService
+    {
+        private const string _HTTP_SERVER_IP = "0.0.0.0";
```

Comment:
We don't need default value as variable because of defaultvalue annotations
https://github.com/aloneguid/config#default-values